e2e_testing/torchscript/elementwise.py

```diff
@@ -515,3 +515,22 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ElementwiseDivScalarModule())
 def ElementwiseDivScalarModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
+
+# ==============================================================================
+class ElementwiseAndIntegerModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.int64, True),
+        ([-1, -1], torch.int32, True),
+    ])
+
+    def forward(self, x, y):
+        return torch.bitwise_and(x, y)
+
+@register_test_case(module_factory=lambda: ElementwiseAndIntegerModule())
+def ElementwiseAndIntegerModule_basic(module, tu: TestUtils):
+    module.forward(torch.randint(-10, 10,(3, 4)), torch.randint(-10, 10,(3, 4)))
```

Comment:
The problem here is torch.randint gives `torch.int64` el-type and you are not typecasting it. Just typecast the second argument i.e., `torch.randint(...).to(torch.int32)` and I guess it will work.

Reply:
Yeah, it worked. Thanks, @pashu123.